WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -695,7 +695,7 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
     func updateForLocalPostsMatchingSearchText() {
         resetTableViewContentOffset()
         updateAndPerformFetchRequest()
-        refreshCachedRowHeightsForTableViewWidth()
+        tableViewHandler.clearCachedRowHeights()
```

Comment:
For this one, I tried a couple variations of this. It seems without calling the `reloadData` method explicitly the results load incorrectly while typing, relative to the searchText. My guess is this is because of the timing/queuing of the `tableView` row changes in the handler.